src/TestInfrastructure/Helpers/MefTestHelpers.cs

```diff
@@ -74,7 +74,7 @@ public static Export CreateExport<T>(object exportInstance, IDictionary<string,
         /// <param name="optionalExports">Any optional exports to include in the composition.</param>
         /// <param name="requiredExports">Any exports required by TTypeToCheck. Optional.</param>
         public static void CheckTypeCanBeImported<TTypeToCheck, TImportType>(
-            object importer,
+            SingleObjectImporter<TImportType> importer,
```

Comment:
I'm going to revert this. If the importer is `object`, then the second type parameter `TImportType` could be removed.